### ISSUE TITLE
Will not load at startup and doesn't capture all downloads from Chrome

### ISSUE THREAD

**Issue by izzyforeal** [subhra74/xdm#291]
Hello, I have selected "launch XDM when system starts" but it doesn't stay checked. Also, I download a lot from rapidgator.net with Google Chrome. In many instances Chrome takes the download instead of XDM. Is there anything I can do about these issues? I am on the latest version and when it works i love it! Thank you.

**Comment by devanks** [comment]
Are you using the xdm extension?

**Comment by izzyforeal** [comment]
Yes I am. The “run at startup” is on the app menu but does not stay on when selected.

**Comment by Bolt2790** [comment]
yeah mine does not run at startup either, I had manually put the exe in startup folder

**Comment by subhra74** [comment]
What os? Windows or linux? What happens if you uncheck and check Run at startup option. Does it work then?

**Comment by izzyforeal** [comment]
Windows 10
Check/Recheck doesn’t work. 
Ive uninstalled and reinstalled as well. 
I did follow your advice and put the app in the startup folder as checking the box on the app does not work. 
Also, with Chrome or Firefox some are downloaded with your app but others are downloaded with the browser. I don’t know why.

**Comment by monkeylove** [comment]
I also have the same problem in Win 10 x64 Home SL. I did the ff.

1. unchecked and checked the startup option and closed the application to make sure that the setting is saved;
2. updated Java and anything else;
3. uninstalled and reinstalled the application (I used the 64
4. did the same things for the two earlier versions of XDM (7.2.10 and 7.2.7).

I also experienced the ff. problems along the way:

1. When I uninstall any of the three versions, a message states that the uninstallation failed, and if I would like to try again. Meanwhile, the application which is minimized in the tray maximizes. When I cancel uninstallation, close the application, and try to uninstall, the same message appears, followed by the application popping up.
2. When I copy the applications shortcut to the start-up folder, it loads when the system starts, but not minimized to the system tray, and even when I configure the properties of the shortcut to start minimized.

I will try version 7.2.8 which is linked to the Sourceforge site. I also noticed that there are two MSIs listed in the 7.2.11 version, and I'll try the second one.
